src/test/java/core/basesyntax/service/read/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service.read.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.service.read.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String DIRECTORY_PATH = "src/test/resources/";
+    private static final String EMPTY_FILENAME = "empty.csv";
+    private static final String NO_VALID_FILENAME = "no_valid.csv";
+    private static final String LINES_1 = "type,fruit,quantity";
+    private static final String LINES_2 = "b,banana,20";
+    private static final String LINES_3 = "b,apple,100";
+    private static final String LINES_4 = "s,banana,103";
+    private static final String LINES_5 = "p,banana,13";
+    private static final String LINES_6 = "r,apple,10";
+    private static final String VALID_FILENAME = "valid.csv";
```

Comment:
there is no sense in this field you can immediately indicate what DIRECTORY_PATH = "src/test/resources/valid.csv"

Reply:
I can't do this, because my readFromFile method takes two parameters. Directory path and filename